### ISSUE TITLE
adapt docs to torch.optim

### ISSUE THREAD

**Issue by Mayukhdeb** [Mayukhdeb/torch-dreams#20]
The `custom_func` definitions in all parts of the documentation should be moved to: `loss = - loss`
